webvh/webvh/did/operations_manager.py

```diff
@@ -0,0 +1,496 @@
+                return did, challenge, domain, expiration
+            else:
+                raise DidCreationError(
+                    "Invalid response from Webvh server requesting identifier"
+                )
+
+    async def _get_or_create_author_key(self, did):
+        async with self.profile.session() as session:
+            try:
+                key_info_author = await MultikeyManager(session).create(
+                    alg="ed25519",
+                    kid=f"{did}#author",
+                )
+            except MultikeyManagerError:
+                key_info_author = await MultikeyManager(session).from_kid(f"{did}#author")
+
+        return key_info_author
+
+    async def _create_author_signed_registration_document(
+        self, did, author_key_info, expiration, domain, challenge
+    ):
+        async with self.profile.session() as session:
+            return await DataIntegrityManager(session).add_proof(
+                DIDDocument(
+                    context=[
+                        "https://www.w3.org/ns/did/v1",
+                        "https://w3id.org/security/multikey/v1",
+                    ],
+                    id=did,
+                    authentication=[author_key_info.get("kid")],
+                    assertion_method=[author_key_info.get("kid")],
+                    verification_method=[
+                        {
+                            "id": author_key_info.get("kid"),
+                            "type": "Multikey",
+                            "controller": did,
+                            "publicKeyMultibase": author_key_info.get("multikey"),
+                        }
+                    ],
+                ).serialize(),
+                DataIntegrityProofOptions(
+                    type="DataIntegrityProof",
+                    cryptosuite="eddsa-jcs-2022",
+                    proof_purpose="assertionMethod",
+                    verification_method=f"did:key:{author_key_info.get('multikey')}#{author_key_info.get('multikey')}",
+                    expires=expiration,
+                    domain=domain,
+                    challenge=challenge,
+                ),
+            )
+
+    async def _wait_for_endorsement(self, did: str):
+        event_bus = self.profile.inject(EventBus)
+        with event_bus.wait_for_event(
+            self.profile, re.compile(rf"^{ENDORSEMENT_EVENT}{did}$")
+        ) as await_event:
+            event = await await_event
+            return await self.finish_create(
+                event.payload.get("document"),
+                state=RegistrationState.FINISHED.value,
+                author_key_info=event.payload.get("author_key_info"),
+                parameters=event.payload.get("metadata", {}).get("parameters"),
+            )
+
+    async def create(self, options: dict):
+        """Register identities."""
+
+        server_url = get_server_info(self.profile)
+
+        namespace = options.get("namespace")
```

Comment:
maybe have a default namespace if non is provided?
`options.get("namespace", "default")`